deploy.sh

```diff
@@ -1,6 +1,6 @@
 set -e
 
-API="https://api.cloud.gov"
+API="https://api.fr.cloud.gov"
```

Comment:
> this is the cloud.gov API url, not the application url

D'oh! DIsregard.

> I don't see a reason to drop the app.cloud.gov binding, though?

Just that it can be potentially confusing to have multiple URLs that can access the same thing. Not a problem on the cloud.gov end!